### ISSUE TITLE
Escape characters in output for dumb terminals

### ISSUE THREAD

**Issue by tprk77** [colcon/colcon-notification#29]
When using Colcon in a dumb terminal, i.e., Emacs, the output contains uninterpreted escape sequences. Ideally, Colcon would detect the environment variable `TERM`, and not output these escape sequences when `TERM=dumb`.

Here's an example of Colcon's output in an Emacs shell (i.e., `M-x shell`):

```
$ colcon build --symlink-install
Starting >>> examples_rclcpp_minimal_action_client
]0;colcon build [0/13 done] [1 ongoing]Starting >>> examples_rclcpp_minimal_action_server
]0;colcon build [0/13 done] [2 ongoing]Starting >>> examples_rclcpp_minimal_client
]0;colcon build [0/13 done] [3 ongoing]Starting >>> examples_rclcpp_minimal_composition
]0;colcon build [0/13 done] [4 ongoing]Starting >>> examples_rclcpp_minimal_publisher
]0;colcon build [0/13 done] [5 ongoing]Starting >>> examples_rclcpp_minimal_service
]0;colcon build [0/13 done] [6 ongoing]Starting >>> examples_rclcpp_minimal_subscriber
]0;colcon build [0/13 done] [7 ongoing]Starting >>> examples_rclcpp_minimal_timer
]0;colcon build [0/13 done] [8 ongoing]Starting >>> examples_rclpy_executors
]0;colcon build [0/13 done] [9 ongoing]Starting >>> examples_rclpy_minimal_client
]0;colcon build [0/13 done] [10 ongoing]Starting >>> examples_rclpy_minimal_publisher
]0;colcon build [0/13 done] [11 ongoing]Starting >>> examples_rclpy_minimal_service
...
```

**Comment by dirk-thomas** [comment]
The `StatusEventHandler` is currently only enabled when `stdout` returns `True` for `isatty` (https://github.com/colcon/colcon-notification/blob/277930f9f5735af3af74fe19bb11ada59a707f94/colcon_notification/event_handler/status.py#L48). In your case this doesn't seem to be specific enough. You can either explicitly disable that event handler by passing `--event-handlers status-` or you could try to improve the logic to handle your case automatically. For that please consider to provide a pull request.

**Comment by dirk-thomas** [comment]
Addressed by #30.
